File: www.app.com/frontend-app/src/app/model/backend/calendar/calendar-dto.ts
import {Addition} from "./addition";
import {Holiday} from "./holiday";
import {Restriction} from "./restriction";
import {Reservation} from "./reservation";

export class CalendarDto {

  public additions: Addition[];

  public holidays: Holiday[];

  public restrictions: Restriction[];

  public reservations: Reservation[];

  public ownReservations: Reservation[];

}

File: www.app.com/core-api/src/main/java/com/base/coreapi/model/calendar/CalendarDTO.java
package com.base.coreapi.model.calendar;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
public class CalendarDTO {

    private List<Addition> additions;

    private List<Holiday> holidays;

    private List<Restriction> restrictions;

    private List<Reservation> reservations;

    private List<Reservation> ownReservations;
}
